config/locales/val/budgets.yml

```diff
@@ -4,9 +4,13 @@ val:
       show:
         title: Els meus vots
         amount_spent: Cost total
-        remaining: "Et queden <span>%{amount}</span> per a invertir."
+        remaining:
```

Comment:
No need to add translations in other languages 😌 

Only English and Spanish are required, the other languages are managed through [Crowdin](https://crowdin.com/project/consul). 

We are still polishing the integration with Crowdin and these changes could create some conflicts 🤔 
Would you mind only adding the translations for English and Spanish? 🙏

Reply:
Sure. I will remove the rest.